app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/views/TermsOfUsePage.scala.html

```diff
@@ -23,14 +23,14 @@
 @(viewModels: List[ViewModel])(implicit request: LoggedInRequest[_], messages: Messages)
 
 @buildStatus(status: String) = @{
-  status.toUpperCase() match {
-    case "EMAIL SENT"                     => <strong class="govuk-tag govuk-tag--blue">{status}</strong>
-    case "IN PROGRESS"                    => <strong class="govuk-tag govuk-tag--blue">{status}</strong>
-    case "PENDING RESPONSIBLE INDIVIDUAL" => <strong class="govuk-tag govuk-tag--blue">{status}</strong>
+  status match {
+    case "EMAIL_SENT"                     => <strong class="govuk-tag govuk-tag--blue">EMAIL SENT</strong>
+    case "REMINDER_EMAIL_SENT"            => <strong class="govuk-tag govuk-tag--blue">REMINDER EMAIL SENT</strong>
+    case "OVERDUE"                        => <strong class="govuk-tag govuk-tag--blue">{status}</strong>
```

Comment:
Are "IN PROGRESS" and "PENDING RESPONSIBLE INDIVIDUAL" supposed to be removed here?

Reply:
Yes, that's deliberate